src/Test/Data/Resource.ts

```diff
@@ -96,7 +96,6 @@ export const response = {
         undefined: 2,
         deploying: 2,
         available: 1,
-        processing_events: 0,
```

Comment:
Maybe add a test case that if it is actually in the response, the application can still handle it?